src/backends/compose/compose_backend.py

```diff
@@ -363,7 +363,7 @@ def add_services(self, tank: Tank, services):
             # it's a git branch, building step is necessary
             repo, branch = tank.version.split("#")
             services[container_name]["image"] = f"{LOCAL_REGISTRY}:{branch}"
-            build_image(repo, branch, LOCAL_REGISTRY, branch, tank.DEFAULT_BUILD_ARGS + tank.extra_build_args, arches="amd64")
+            build_image(repo, branch, LOCAL_REGISTRY, branch, tank.DEFAULT_BUILD_ARGS + tank.build_args, arches="amd64")
```

Comment:
FWIW `tank` properties names MUST now exactly match the grpah schema, as they are parsed and set according to it.

Should probably document that...